pyproject.toml

```diff
@@ -0,0 +1,132 @@
+[build-system]
+requires = [
+    "setuptools>=62",
+    "wheel",
```

Comment:
You don't have to specify `wheel` since it is automatically added by setuptools :)

Reply:
I don't think so, since it is not mentioned in the dependencies of setuptools and also `pip show setuptools` doesn't list it as a requirement. 
On the [setuptools website](https://setuptools.pypa.io/en/latest/setuptools.html#setup-cfg-only-projects) there is also an example given with it:
```toml
[build-system]
requires = [
  "setuptools >= 40.9.0",
  "wheel",
]
build-backend = "setuptools.build_meta"
```
But still, it is not "really" required to install spotpy, but to build wheels. In [this stackoverflow post](https://stackoverflow.com/questions/61301316/how-and-who-should-specify-wheel-as-a-python-package-dependency) it is also stated as a quasi-standard to add it here. So I would keep it.

Reply:
I will update the setuptools website to make sure wheels is not mention in `requires`, sorry for that.

Setuptools will add `wheel` to the isolated build environment (this os probably the reason why you don't see it when you run `pip list`, once the build is done the environmentis deleted and `wheel` does not stick around).

Reply:
> I will update the setuptools website to make sure wheel is not mention in requires, sorry for that.

Done. I have removed `wheel` in the mentioned example and any other that I could find. Please let me know if there is anything left.

Reply:
> Good to know. Is that written anywhere in the setuptools documentation that wheel will be installed for isolated builds?

It does not have to be, right? Setuptools has to ensure all the dependencies itself requires for isolated builds are installed in the temporary virtual environment. This should be transparent for the end users.

Currently, the implementation uses `get_requires_for_build_wheel` (from PEP 517), but in the future this might be different (e.g. wheel can be directly vendored, or setuptools might decide to implement `bdist_wheel` itself). We can consider it as an implementation detail in the context of PEP 517/518.

(The users would still need to add entries to `[build-system] requires` if they need something *extra*, like `setuptools-scm`, `setuptools-golang`, `Cython`).

Reply:
@abravalheri thank you for this input, that makes the build even easier.

And referring to:
> moved to src/ based package structure for easier testing and so setuptools finds the correct package data

@MuellerSeb looking randomly at some other packages (e.g. [setuptools](https://github.com/pypa/setuptools)), I did not find the /src folders. Personally, I am not a big fan of having this additional /src folder. But maybe I do not get the benefits of this structure. What's your thought on this?

Reply:
Ok cool, with that background info it actually makes sense to me to have the extra `src` folder :)